helper.js

```diff
@@ -31,6 +31,16 @@ module.exports = {
         window.localStorage.setItem('nightwind-mode', 'light');
     }
   },
+  
+  setDarkMode: () => {
+    document.documentElement.classList.add('dark');
+    window.localStorage.setItem('nightwind-mode', 'dark');
+  },
+
+  setLightMode: () => {
+    document.documentElement.classList.remove('dark');
+    window.localStorage.setItem('nightwind-mode', 'light');
+  },
 
```

Comment:
Could you turn them into a single function, taking a boolean argument to pick the theme to switch to? (Similar to original #19 proposal)